### PR TITLE
add icon for brand icon

### PR DESCRIPTION
**Pull request by zikowang**
| Status  | Type  |
| :---: | :---: |
| Ready| Feature |

## Description
Add brand icon

## How Has This Been Tested?
locally

## Screenshots / GIFs (if appropriate):

image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Enhancement (non-breaking change which cleans up / improves existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] I have read the **CONTRIBUTING** document.
- [x] I have considered the accessibility of my changes (i.e. did I add proper content descriptions to images, or run my changes with talkback enabled?)
- [x] I have documented my code if needed

### CODE REVIEW

**Review by luwol03 — APPROVED** [review]

File: src/App.jsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ import Settings from "./screens/Settings/Settings.jsx";
 import { signIn, signOut } from "./redux/Actions/login.js";
 import { getProfile } from "./utils/api.js";
 
-import Image from "./images/vocascan-logo.png";
+import Image from "./images/logo/vocascan-logo.png";
```
Comment (luwol03) [inline src/App.jsx:16]: shouldn't we use the svg here?

File: src/Components/Nav/TopNav.jsx [hunk 2]
```diff
@@ -9,7 +9,7 @@ import PersonIcon from "@material-ui/icons/Person";
 
 import { signOut } from "../../redux/Actions/login.js";
 
-import LogoRound from "../../images/logo-round.svg";
+import LogoRound from "../../images/logo/logo-round.svg";
```
Comment (luwol03) [inline src/Components/Nav/TopNav.jsx:12]: shouldn't we name this `transparent-round.svg` and `transparent-react.svg` for consistency?

File: src/App.jsx (hunk 1 above)
Comment (zikowang) [reply src/App.jsx]: yes, svg is better here

File: src/Components/Nav/TopNav.jsx (hunk 2 above)
Comment (zikowang) [reply src/Components/Nav/TopNav.jsx]: good point

File: src/App.jsx [hunk 3]
```diff
@@ -13,7 +13,7 @@ import Settings from "./screens/Settings/Settings.jsx";
 import { signIn, signOut } from "./redux/Actions/login.js";
 import { getProfile } from "./utils/api.js";
 
-import Image from "./images/logo/vocascan-logo.png";
+import Image from "./images/logo/transparent-round.svg";
```
Comment (luwol03) [inline src/App.jsx:16]: can we use the color version for that?

Comment (zikowang) [reply src/App.jsx]: Of course, my bad!

**Review by luwol03 — APPROVED** [review]

**Pull request merged**